src/subcommands/subcommand.go

```diff
@@ -84,16 +84,12 @@ func baseHelp() {
 	fmt.Println("\nUse 'semantifly <subcommand> --help' for more information about a specific subcommand.")
 }
 
-func setupSemantifly() {
-	semantifly_dir := flag.String("semantifly_dir", os.ExpandEnv("$HOME/.semantifly"), "Where to read existing semantifly data, and write new semantifly data.")
-	semantifly_index := flag.String("semantifly_index", "default", "By default, semantifly writes data to the 'default' subdir of the configured semantifly_dir. Setting this to a value other than 'default' allows for multiple indices on the same local machine.")
-
-	createDirectoryIfNotExists(*semantifly_dir)
-	index_path := path.Join(*semantifly_dir, *semantifly_index)
+func setupSemantifly(semantifly_dir string, semantifly_index string) string {
+	createDirectoryIfNotExists(semantifly_dir)
```

Comment:
We should create these directories later, once we know the actual index path, because there's no need to create these if the user specifies an alternative directory + if they do specify an alternative directory, we may need to create that instead

Reply:
done